tests/module/animation/test_animation.py

```diff
@@ -12,26 +12,21 @@
 def test_animation_forbidden_run_time(run_time):
     manager = Manager(Scene)
     test_scene = manager.scene
-    with pytest.raises(ValueError, match="Please set the run_time to be positive"):
+    with pytest.raises(
+        ValueError, match="Please set the run_time to a positive number."
+    ):
         test_scene.play(FadeIn(None, run_time=run_time))
 
 
 def test_animation_run_time_shorter_than_frame_rate(manim_caplog, config):
     manager = Manager(Scene)
     test_scene = manager.scene
     test_scene.play(FadeIn(None, run_time=1 / (config.frame_rate + 1)))
-    assert (
-        "Original run time of FadeIn(Mobject) is shorter than current frame rate"
-        in manim_caplog.text
-    )
+    assert "too short for the current frame rate" in manim_caplog.text
 
 
-@pytest.mark.parametrize("frozen_frame", [False, True])
-def test_wait_run_time_shorter_than_frame_rate(manim_caplog, frozen_frame):
+def test_wait_run_time_shorter_than_frame_rate(manim_caplog):
     manager = Manager(Scene)
     test_scene = manager.scene
-    test_scene.wait(1e-9, frozen_frame=frozen_frame)
```

Comment:
We should probably get `frozen_frame` working instead of removing it from the test?

Reply:
`frozen_frame` was no longer among the parameters of the new `Scene.wait()`, so I assumed that it would no longer be needed.

I can put it there again, if it's necessary.

Reply:
Yeah, that might be my fault actually - I think I might have wanted to replace it with `wait_until`?
I guess for now until `wait_until` works this is fine.